app/src/pages/lloydGeorgeUploadPage/LloydGeorgeUploadPage.tsx

```diff
@@ -148,12 +164,20 @@ function LloydGeorgeUploadPage() {
                     progress: 100,
                 });
             } catch (e) {
+                window.clearInterval(intervalTimer);
```

Comment:
``
```suggestion
                window.clearInterval(updateStateInterval);
```

For this line, I think we should call clearInterval with updateStateInterval, not intervalTimer
Because line 144, 145 and line 167 are in the same scope, the new `intervalTimer` state set in line 145 would not be reflected in line 167.

Reply:
agree